components/Admin/AdminButton.tsx

```diff
@@ -82,19 +157,31 @@ function AdminButton({
     }    
   }
 
+  const handleFeatureCheckbox = (id, state, event) => {
+    event.stopPropagation();
+    const isSelected = makeFeaturedChoices.includes(id);
+    if (isSelected) {
+      const newChoices = makeFeaturedChoices.filter(c => c !== id)
+      setMakeFeaturedChoices(newChoices);
```

Comment:
newChoices is instantiated every time the function is called. No need to store as a var. Might be easier to read in 1 line
```suggestion
      setMakeFeaturedChoices(makeFeaturedChoices.filter(c => c !== id));
```

Reply:
👍